### ISSUE TITLE
In which Training step do you use HH-RLHF and SHP datasets?

### ISSUE THREAD

**Issue by richhh520** [IBM/SALMON#5]
In which Training step do you use HH-RLHF and SHP datasets?
Thanks for your help.

**Comment by Edward-Sun** [comment]
Hi @richhh520 

They are used in the first step of training the reward model, i.e., [preference model pre-training](https://github.com/IBM/SALMON/blob/main/training/step2_rm_training/scripts/train_reward_model_70b_qlora_pmp.sh)
